classes/ProductSale.php

```diff
@@ -132,6 +132,14 @@ public static function getBestSales($id_lang, $page_number = 0, $nb_products = 1
         }
 
         if ($final_order_by != 'price') {
+            if (!Validate::isOrderBy($order_by)) {
+                $order_by = 'cp.position';
+            }
+
+            if (!Validate::isOrderWay($order_way)) {
+                $order_way = 'ASC';
+            }
```

Comment:
@thierrymarianne  How did you chose to " position " and " asc " ?
Previously, in the function, there is default $order_by (quantity) & $order_way (desc) if the fields were empty. It's not better to take this ?

```
if (is_null($order_by)) {
    $order_by = 'quantity';
    $order_table = 'ps';
}

if (is_null($order_way) || $order_by == 'sales') {
    $order_way = 'DESC';
}
```